### ISSUE TITLE
Issue while Building the RV32I Toolchain

### ISSUE THREAD

**Issue by shuvamnandi** [cliffordwolf/picorv32#29]
Hi, 
I was trying to build the RV32I toolchain and got an error at this step `../configure --with-arch=RV32I --prefix=/opt/riscv32i`:

```../configure --with-arch=RV32I --prefix=/opt/riscv32i
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for grep that handles long lines and -e... /bin/grep
checking for fgrep... /bin/grep -F
checking for grep that handles long lines and -e... (cached) /bin/grep
checking for bash... /bin/bash
checking for __gmpz_init in -lgmp... yes
checking for mpfr_init in -lmpfr... yes
checking for mpc_init2 in -lmpc... yes
checking for curl... /usr/share/centrifydc/bin/curl
checking for wget... /usr/bin/wget
checking for ftp... /usr/bin/ftp
configure: error: Unknown arch
```

Did anyone else also face this problem?

**Comment by cliffordwolf** [comment]
Oops. I've updated to a new toolchain version yesterday and forgot to update that bit of documentation. The arguments for `--with-arch=` is now lowercase. Fixed README in e9b6bcf. Thanks for reporting this issue.

**Comment by shuvamnandi** [comment]
Yes, it works after changing the argument. Thanks!
